filebeat/module/traefik/access/config/traefik-access.yml

```diff
@@ -7,9 +7,9 @@ exclude_files: [".gz$"]
 
 processors:
 - dissect:
-    tokenizer: '%{traefik.access.remote_ip} - %{traefik.access.user_name} [%{traefik.access.time}]
-    "%{traefik.access.method} %{traefik.access.url} HTTP/%{traefik.access.http_version}"
-    %{traefik.access.response_code} %{traefik.access.message}'
+    tokenizer: '%{traefik.access.remote_ip} - %{user.name} [%{traefik.access.time}]
```

Comment:
Is remote_ip `source.ip`?

Reply:
is access.time the `@timestamp`?

Reply:
It eventually becomes the timestamp, once parsed by IN, yes. 

remote_ip is a host. So in Ingest Node I try to parse it as an IP then store in `source_ecs.ip`, if that fails, it falls back to `source.domain`.